thunder/executors/nvfuserex_impl.py

```diff
@@ -1759,6 +1763,15 @@ def clone(a: TensorProxy, *, fd: FusionDefinition, lc_to_nv_map: dict) -> Any:
 
 register_supported(PrimIDs.CLONE, clone, _elementwise_unary_check)
 
+# update_aliases is disabled.  nvfuser does not support it.
+# def update_aliases(aliases: tuple[TensorProxy], *, fd: FusionDefinition, lc_to_nv_map: dict) -> Any:
+#     nvaliases = tuple(getnv(alias, fd, lc_to_nv_map) for alias in aliases)
+#     return tuple(fd.ops.set(nvalias) for nvalias in nvaliases)
+
+
+# register_supported(PrimIDs.UPDATE_ALIASES, update_aliases, _elementwise_nnary_check)
```

Comment:
It'd be great to put TODO comment and/or reference of issue (if exists) here

Reply:
There is no issue yet, in part because the requirements for nvfuser are not firm. But will add a TODO.